### PR TITLE
Clean up dependencies

### PR DESCRIPTION
**Pull request by XhmikosR**
Remove unused ones and move devDependencies to devDependencies.

Unused deps:

* require-dir

devDependencies:

* chokidar
* st

Also, regenerate package-lock.json.

### CODE REVIEW

**Comment by XhmikosR** [comment]
This needs to wait until the other dep PRs are merged, namely #2611 and #2614

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**
